MANIFEST.in

```diff
@@ -1,6 +1,7 @@
-include ez_setup.py
```

Comment:
ez_setup was nice to bootstrap before setuptools or pip was included in the python installs.  I think in 2.7 and onwards its not really needed as both come with the base install.  If you still want easy_install functionality, it could be added back just untested on my part.  You can still install with `pip install PyIBIS-AMI` or `conda install PyIBIS-AMI`.